vaadin-grid-flow-integration-tests/src/main/java/com/vaadin/flow/component/grid/it/dataview/ItemCountCallbackGridPage.java

```diff
@@ -20,7 +20,7 @@
 import com.vaadin.flow.router.Route;
 
 @Route("row-count-callback/:count?([0-9]{1,9})")
```

Comment:
Please just rename the route paths too to avoid confusion later on for all the tests from "row-count..." to "item-count..."

Reply:
done